tests/testthat/test-mock-repos.R

```diff
@@ -104,3 +104,21 @@ test_that("repos, some basics", {
   )
   expect_equal(res[[1]], "")            # TODO: better return value here?
 })
+
+test_that("repo raw files", {
+  skip_if_offline()
+  skip_on_cran()
+  skip("needs mocking")
+
+  res <- gh(
+    "GET /repos/:owner/:repo/contents/:path",
+    owner = "r-lib",
+    repo = "gh",
+    path = "DESCRIPTION",
+    .send_headers = c(Accept = "application/vnd.github.v3.raw"),
```

Comment:
I wonder if we could just always send this header? Is it s problem for the "normal" end points that return JSON? Or is it a good idea to make the user think about the output format?

Reply:
I just tested this on an endpoint that doesn't have a raw response, and it returned JSON fine. 
 But the user may not want the raw version when it is available.  The default for file contents and blobs is to return JSON with the file contents as a base64 field.  If you always send the header, it should probably be the JSON one.  The other options available for some endpoints (`diff`, `patch`, `sha`) are all handled fine as text.

Reply:
Got it.